### PR TITLE
Issue #SB-24699 fix: Observation without rubric link

### PR DESCRIPTION
**Pull request by vishwanath1004**

### CODE REVIEW

File: src/app/manage-learn/questionnaire/questionnaire.page.ts [hunk 1]
```diff
@@ -96,39 +102,50 @@ export class QuestionnairePage implements OnInit, OnDestroy {
...
+        this.getQuestions(data);
+      })
+    }
+  }
 
-        this.selectedEvidenceId = currentEvidences[this.selectedEvidenceIndex].externalId;
-        this.localImageListKey = 'images_' + this.selectedEvidenceId + '_' + this.submissionId;
-        this.isViewOnly = !currentEvidences[this.selectedEvidenceIndex]['startTime'] ? true : false;
+  getQuestions(data){
+    this.selectedSectionIndex =0;
```
Comment (kiranharidas187) [inline src/app/manage-learn/questionnaire/questionnaire.page.ts:117]: repeating 2 times?

File: src/app/manage-learn/questionnaire/questionnaire.page.ts [hunk 2]
```diff
@@ -79,6 +79,12 @@ export class QuestionnairePage implements OnInit, OnDestroy {
       this.schoolName = params.schoolName;
     });
 
+    this.extrasState = this.router.getCurrentNavigation().extras.state;
+    if (this.extrasState) {
```
Comment (kiranharidas187) [inline src/app/manage-learn/questionnaire/questionnaire.page.ts:83]: is this required?

File: src/app/manage-learn/questionnaire/questionnaire.page.ts [hunk 3]
```diff
@@ -79,6 +79,12 @@ export class QuestionnairePage implements OnInit, OnDestroy {
...
+      // this.data = extrasState.data;
```
Comment (kiranharidas187) [inline src/app/manage-learn/questionnaire/questionnaire.page.ts:84]: where are we setting view only mode as true??

File: src/app/manage-learn/questionnaire/questionnaire.page.ts (hunk 1 above)
Comment (vishwa1004) [reply src/app/manage-learn/questionnaire/questionnaire.page.ts:113]: this.selectedSectionIndex =0; and this.selectedEvidenceIndex =0; are different

**Pull request closed without merging**